packages/lib/dayjs/index.ts

```diff
@@ -79,13 +79,19 @@ export const formatToLocalizedDate = (
  * given Date object and locale. Undefined values mean the defaults
  * associated with the browser's current locale will be used.
  */
-export const formatToLocalizedTime = (
-  date: Date | Dayjs,
-  locale: string | undefined = undefined,
-  timeStyle: Intl.DateTimeFormatOptions["timeStyle"] = "short",
-  hour12: Intl.DateTimeFormatOptions["hour12"] = undefined,
-  timeZone?: string
-) => formatLocalizedDateTime(date, { timeStyle, hour12, timeZone }, locale);
+export const formatToLocalizedTime = ({
+  date,
+  locale = undefined,
+  timeStyle = "short",
+  hour12 = undefined,
+  timeZone,
```

Comment:
Would it make sense to make timeZone a required field, so that goign forward this timeZone isn't accidentally missed